python/lsst/obs/subaru/gen3/hsc/instrument.py

```diff
@@ -79,18 +87,8 @@ def register(self, registry):
                 # HSC doesn't have rafts
                 raft=None
```

Comment:
Oh, HSC does have rafts as well. `1` and `2` and `ObservationInfo` knows about them. @TallJimbo why can't we add them?

Reply:
I don't mind if we do.  I'm not sure we have a use case, but I guess there's a chance it might be useful, and not declaring any rafts doesn't have any chance of being useful.

Reply:
What is meant by "raft" these days? My understanding is that the HSC focal plane consists of a single Invar surface, but the CCDs are divided into two electronic backends. So does "raft" refer to a mechanical or electronic separation?

Reply:
In astro_metadata_translator I call them "detector groups". THere is code in obs_subaru bin.src that uses the term raft so I assumed obs_subaru would like to have access to the "raft/detector group" in gen 3.

Reply:
Can we deal with that on a separate ticket, please?

Reply:
Similarly to my comment on the obs_decam PR, the HSC things are not `rafts`, they are groups of detectors. Any such definition of a grouping of detectors should be dealt with on another ticket.